extensions/wikia/Recirculation/templates/client/incontent.mustache

```diff
@@ -0,0 +1,14 @@
+<div class="recirculation-horizontal recirculation-incontent">
+	<h3>{{ title }}</h3>
+
+	<div class="items">
+		{{#items}}
+		<div class="item">
+			<a title={{ title }} href="{{ url }}">
+				<div class="thumbnail"><img src="{{ thumbnail }}" /></div>
```

Comment:
I don't think we're using the self-closing XHTML-style tags anymore. Plus I think `alt` property is still required for img tags.